libs/parallelism/algorithms/include/hpx/parallel/algorithms/search.hpp

```diff
@@ -15,7 +16,7 @@
 #include <hpx/execution/algorithms/detail/predicates.hpp>
 #include <hpx/executors/execution_policy.hpp>
 #include <hpx/parallel/algorithms/detail/dispatch.hpp>
-#include <hpx/parallel/algorithms/for_each.hpp>
+#include <hpx/parallel/container_algorithms/detail/search.hpp>
```

Comment:
This is a bit unusual as normally the container versions depend on the non-container versions. Would it be possible to move this header here instead?

Reply:
Yes no problem, I put it there as I had to do a sentinel adaptation and I thought it was related to container_algorithms.
Will do the change ;)